bindings/python/src/ffi.rs

```diff
@@ -0,0 +1,10 @@
+use std::os::raw::c_char;
+
+/// ob_type is not exposed by default as is needed to check if something inherits from a enum
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct PyTypeObjectSubObType {
```

Comment:
As it corresponds to the `PyTypeObject` struct in CPython, it might be better to keep the original name, so we know that it is our "re-definition" (because the one in PyO3 is opaque and we can't get the `ob_type` field from it)

Reply:
I originally had it named this way but changed the name so I didnt have two `PyTypeObject` (Kinda confusing as to which one you are using) will switch it back